uint8 GoalPosition
---
bool Success
